### PR TITLE
Homepage

### PR DESCRIPTION
**Pull request by mrCherry97**
**Description**
The first implementation of HomePage with navigation and routing to documentation.

Changes proposed in this pull request:

- HomePage
- NavBar
- Header
- Routing

### CODE REVIEW

File: docs/.vuepress/theme/layouts/HomePage/Header.vue [hunk 1]
```diff
@@ -0,0 +1,280 @@
...
+                </PopoverButton>
+              </div>
+            </div>
+
+            <div class="mt-6">
+              <nav class="grid gap-y-8 text-center">
+                <a href="#" class="m-0 text-lg font-medium"> Docs </a>
+                <a href="#" class="m-0 text-lg font-medium"> Blog </a>
+                <a href="#" class="m-0 text-lg font-medium"> Community </a>
+                <a href="#"> Slack </a>
```
Comment (Lyczeq) [inline docs/.vuepress/theme/layouts/HomePage/Header.vue:155]: Do these spaces need to be there?

File: docs/.vuepress/theme/layouts/HomePage/Header.vue [hunk 2]
```diff
@@ -0,0 +1,280 @@
...
+  ChartBarIcon,
+  CursorClickIcon,
+  MenuIcon,
+  PhoneIcon,
+  PlayIcon,
+  RefreshIcon,
+  ShieldCheckIcon,
+  SupportIcon,
+  ViewGridIcon,
+  XIcon,
```
Comment (Lyczeq) [inline docs/.vuepress/theme/layouts/HomePage/Header.vue:179]: Probably some of those imports are not necessary.

File: docs/.vuepress/theme/layouts/HomePage/Header.vue [hunk 3]
```diff
@@ -0,0 +1,280 @@
...
+    name: 'Community',
+    description: "Connect with third-party tools that you're already using.",
+    href: '#',
+  },
+  {
+    name: 'Slack',
+    description: 'Build strategic funnels that will drive your customers to convert',
+    href: '#',
+  },
+];
```
Comment (Lyczeq) [inline docs/.vuepress/theme/layouts/HomePage/Header.vue:204]: That array isn't used anywhere.

File: docs/.vuepress/theme/layouts/HomePage/Header.vue [hunk 4]
```diff
@@ -0,0 +1,280 @@
...
+const callsToAction = [
+  { name: 'Watch Demo', href: '#', icon: PlayIcon },
+  { name: 'Contact Sales', href: '#', icon: PhoneIcon },
+];
+const resources = [];
+const recentPosts = [
+  { id: 1, name: 'Boost your conversion rate', href: '#' },
+  { id: 2, name: 'How to use search engine optimization to drive traffic to your site', href: '#' },
+  { id: 3, name: 'Improve your customer experience', href: '#' },
+];
```
Comment (Lyczeq) [inline docs/.vuepress/theme/layouts/HomePage/Header.vue:214]: As I can see, those arrays aren't used as well. If that's right, please remove them also in the setup method.

File: docs/.vuepress/theme/layouts/HomePage/HomePage.vue [hunk 5]
```diff
@@ -0,0 +1,61 @@
...
+
+<!-- Add "scoped" attribute to limit CSS to this component only -->
+<style scoped>
+/* p {
+  margin-top: 180 px;
+  margin-bottom: 39 px;
+  font-size: 32px;
+  font-weight: 400;
+  line-height: 1.5;
+} */
```
Comment (Lyczeq) [inline docs/.vuepress/theme/layouts/HomePage/HomePage.vue:60]: Is it needed if it's commented?

File: tailwind.config.js [hunk 6]
```diff
@@ -1,10 +1,24 @@
...
-      './docs/.vuepress/**/*.ts',
-    ],
+    content: ['./docs/.vuepress/**/*.js', './docs/.vuepress/**/*.vue', './docs/.vuepress/**/*.ts'],
+  },
+  darkMode: false, // or 'media' or 'class'
+  theme: {
+    fill: (theme) => ({
+      red: theme('colors.red.700'),
+      green: theme('colors.green.700'),
+      blue: theme('colors.blue.700'),
```
Comment (Lyczeq) [inline tailwind.config.js:11]: As I observed, we don't use green and red colours on the website. I think we can remove those two attributes.

**Review by dariadomagala — APPROVED** [review]

**Pull request merged**